### DISCUSSION TITLE
How to get current color scheme?

### DISCUSSION THREAD

**Discussion by nighthawk469** [wezterm/wezterm#5024] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'm using this first script to use a random color scheme on each terminal window. When I find a nice theme, I want to know what it is called. How can I get the current scheme name ?
https://wezfurlong.org/wezterm/config/lua/wezterm/get_builtin_color_schemes.html

**Comment by wez** [comment, marked as answer]
use CTRL-SHIFT-L to open the debug overlay, then type in `window:get_config_overrides().color_scheme`
